requirements/postgres.txt

```diff
@@ -0,0 +1,2 @@
+# dependencies needed to integrate with Postgres
+psycopg2-binary
```

Comment:
Is this used and still required?

Reply:
This is not required yet. SQLAlchemy will use the psycopg2 library for driving connections with Postgres. I've included it as a foundational commit to, because I was using it to verify the skipping behavior of `@requires_postgres` test decorator.